docs/sql-guide/functions/function-flatten.md

```diff
@@ -0,0 +1,76 @@
+---
+title: FLATTEN()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# FLATTEN() function
+
+The `FLATTEN()` function is used when a query wants to get distinct or group on individual members of [IDSET](/docs/sql-guide/data-types/data-type-idset) and [STRINGSET](/docs/sql-guide/data-types/data-type-stringset) columns
+
+## Syntax
+
+```
+flatten(column)
+```
+
+## Arguments
+
+| Argument | Data type | Description | Required? | Further information |
+|---|---|---|---|---|
+| `column` | IDSET/STRINGSET | [IDSET](/docs/sql-guide/data-types/data-type-idset) and [STRINGSET](/docs/sql-guide/data-types/data-type-stringset) columns | Yes | This should only be used with `GROUP BY` queries |
+
+## Returns
+
+| Data type | Value |
+|---|---|
+| `ID`/`STRING` | individual values of passed column |
+
+## Additional information
+
+* This can only be used in the [WITH clause](/docs/sql-guide/statements/statement-select/#with_clause)
+* This should only be used with `GROUP BY` queries
+
+## Examples
+
+### STRINGSET with flatten() that counts individual values
+```sql
```

Comment:
I'd suggest separating these and stating what's what.

### Create table and insert data to query

```
create table...
insert into...
```
### STRINGSET with flatten()...


### STRINGSET without flatten()...

Reply:
While I'm with you, that's not consistent with all of our function docs: https://docs.featurebase.com/docs/sql-guide/functions/function-datetimediff/. I'd rather be consistent here. I think there is a standardization need AFTER rewrites are finished.